src/components/project/project.repository.ts

```diff
@@ -282,10 +282,10 @@ export class ProjectRepository extends CommonRepository {
       .apply(this.privileges.for(session, IProject).filterToReadable())
       .apply(
         sorting(IProject, input, {
-          sensitivity: (query) =>
+          engagements: (query) =>
             query
-              .apply(matchProjectSens('node'))
-              .return<{ sortValue: string }>('sensitivity as sortValue'),
```

Comment:
We still need to keep this custom sorting for sensitivity. However, I'm realizing this doesn't actually sort correctly.
We need to call convert to a sens rank
```ts
.return<{ sortValue: string }>(`${rankSens('sensitivity')} as sortValue`),
```

Reply:
I was actually wrong here.
This query just customized how the sensitivity value was retrieved.
The [field still had a DbSort transformer](https://github.com/SeedCompany/cord-api-v3/blob/f67c7cc8f7c9e121b76852071f7aa013e4b302ca/src/common/sensitivity.enum.ts#L18-L18) which was applied in `sorting()`'s next step.
So no change actually needed here.